.github/workflows/integration.yml

```diff
@@ -90,6 +90,47 @@ jobs:
         SINGLE_COMMIT: True
         COMMIT_MESSAGE: build from commit ${{ github.sha }}
 
+  babel:
+    name: Babel
+    runs-on: ubuntu-20.04
+    if: ${{ github.repository_owner == 'searxng' && github.ref == 'refs/heads/master' }}
+    needs:
+      - python
+      - themes
+      - documentation
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        persist-credentials: false
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
+        architecture: 'x64'
+    - name: Update transations
+      id: update
+      continue-on-error: true
+      run: |
+        pip install babel jinja2
+        searx_extra/update/update_translations.sh
```

Comment:
This code should be executed in the virtualenv (./local/py3) / not in the environment from the OS.

Reply:
Note: the CI runs this code in a throw away VM: everything runs as root.

Reply:
Anyway, we do all build task in a dedicated environment and this is the environment in ./local/py3

By example: I tried to update translations and it fails on my desktop PC.

With commit 1a66cad all pybabel commands are executed in this environment by  the leading `pyenv.cmd pybabel`